scripts/plot.py

```diff
@@ -1,8 +1,8 @@
-import matplotlib.pyplot as plt
-import numpy as np
+import sys
```

Comment:
If there are linting requirements for ordering of imports we should certainly provide some autoformatter assistance in the near future

Reply:
```console
scripts/plot.py:3:0: C0411: standard import "os.path" should be placed before third party imports "matplotlib.pyplot", "numpy" (wrong-import-order)
```
Yes, system imports should be above third party packages